defaults/main/sshd.yml

```diff
@@ -43,17 +47,16 @@ sshd_ignore_user_known_hosts: "yes"
 sshd_kerberos_authentication: "no"
 sshd_kex_algorithms: >-
   curve25519-sha256@libssh.org,
-  ecdh-sha2-nistp521,
-  ecdh-sha2-nistp384,
-  ecdh-sha2-nistp256,
-  diffie-hellman-group-exchange-sha256
+  curve25519-sha256,
+  diffie-hellman-group-exchange-sha256,
+  diffie-hellman-group16-sha512,
+  diffie-hellman-group18-sha512
 sshd_login_grace_time: 20
 sshd_log_level: VERBOSE
 sshd_macs: >-
   hmac-sha2-512-etm@openssh.com,
   hmac-sha2-256-etm@openssh.com,
-  hmac-sha2-512,
-  hmac-sha2-256
```

Comment:
The default for macs, encrypt-and-MAC, isn't very good and was suggested to be removed.